sdk/communication/azure-communication-phonenumbers/test/_shared/utils.py

```diff
@@ -32,3 +32,13 @@ def get_http_logging_policy(**kwargs):
         }
     )
     return http_logging_policy
+
+def get_header_policy(**kwargs):
+    header_policy = HeadersPolicy(**kwargs)
+
+    import os
+    useragent = os.getenv("AZURE_USERAGENT_OVERRIDE")
```

Comment:
Where the `AZURE_USERAGENT_OVERRIDE` will be defined? Shouldn't the definition of the variable be part of this PR?

Reply:
The value of this variable does not belong to the scope of this PR, and should be defined in a per environment basis